apptools/preferences/tests/test_preferences_helper.py

```diff
@@ -192,7 +192,7 @@ class AcmeUIPreferencesHelper(PreferencesHelper):
             width       = Int(50)
             ratio       = Float(1.0)
             visible     = Bool(True)
-            description = Unicode(u'description')
+            description = Str(u'description')
```

Comment:
We could go further and drop the `u` prefixes, but that's probably best done in a separate PR.